src/cljs/main/broadfcui/page/workspace/analysis/igv.cljs

```diff
@@ -10,61 +11,58 @@
    [broadfcui.utils.user :as user]
    ))
 
-(defonce ^:private igv-styles-loaded? (atom false))
-
-(defn- load-igv-styles []
-  (.. (js/$ "head") ; IGV uses jQuery, so I'm using it here too.
-      (append "<link rel=\"stylesheet\" type=\"text/css\" href=\"https://igv.org/web/release/1.0.1/igv-1.0.1.css\">"))
-  (reset! igv-styles-loaded? true))
-
 (defn- options [tracks token]
   (clj->js
-   {:genome "hg19"
-    :trackDefaults {:palette ["#00A0B0" "#6A4A3C" "#CC333F" "#EB6841"]
-                    :bam {:coverageThreshold 0.2
-                          :coverageQualityWeight true}}
```

Comment:
I've removed as many of our overrides as possible, so we are using all the new/improved IGV defaults.